### ISSUE TITLE
Bug: EBS volumes with custom CMK shows it's using the default *aws/ebs* key.

### ISSUE THREAD

**Issue by frezbo** [rexray/rexray#1241]
# Summary
Please enter a summary of the issue here.
EBS volumes with custom CMK shows it's using the default *aws/ebs* key.

# Bug Reports
EBS volumes with custom CMK shows it's using the default *aws/ebs* key, but `aws describe volumes` shows its using custom CMK. A quick glance at the code does look good. I have confirmed the it shows the correct key in console when using terraform.

## Version
`0.11.1`

## ScreenShots

![pasted image at 2018_06_21 11_20 am](https://user-images.githubusercontent.com/18496730/41716175-3836ab7c-7573-11e8-8cbc-16ba4b2cc4c1.png)

![pasted image at 2018_06_21 11_22 am](https://user-images.githubusercontent.com/18496730/41716182-4041f632-7573-11e8-98e2-d863ce763268.png)

![pasted image at 2018_06_21 10_53 am](https://user-images.githubusercontent.com/18496730/41716191-49d91c84-7573-11e8-8dbb-8ef8dc133237.png)

**Comment by clintkitson** [comment]
Thanks for the repo @frezbo. Just to be clear, this is a reporting problem only? Functionality works but it is reporting back the wrong CMK?

**Comment by frezbo** [comment]
@clintkitson Yes you are right, but this does not happen with terraform. Sorry for the late reply, I did not get a github notification for this.
